OSSSpeechKit/Classes/OSSSpeech.swift

```diff
@@ -280,6 +280,21 @@ public class OSSSpeech: NSObject {
         speak()
     }
     
+    /// Stop speaking text
+    public func stopSpeaking() {
+        speechSynthesizer.stopSpeaking(at: .immediate)
```

Comment:
Couple of items of feedback
1. Because `stopSpeaking` returns a BOOL value, I would have this function return a BOOL  
2. I'd recommend checking if there is something speaking for first via 
```
    if speechSynthesizer.isSpeaking { return false }
```
3. In the startRecording OR startSpeaking function,  we need stopSpeaking as this may potentially throw an exception; I am not entirely sure whether Apple refreshes the instance each time `startSpeaking` is called so if you have it paused, request it speak something, then request it continue later on, I don't know what would happen, so it'd be safer to call `speechSynthesizer.stopSpeaking(at: immediate)` whenever a speak or record event occurred. 

From Apple Docs - [isSpeaking](https://developer.apple.com/documentation/avfaudio/avspeechsynthesizer/1619680-isspeaking)